scripts/run-tests.sh

```
@@ -1,23 +1,37 @@
 #bin/bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)